### PR TITLE
Add hook for AnyIO

### PR DESCRIPTION
**Pull request by agronholm**

### CODE REVIEW

**Comment by agronholm** [comment]
The Travis failures don't seem to have any connection to my hook. Why keep Travis and Appveyor checks when you have Github Actions?

File: news/22.new.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Add hook for ``AnyIO`` which dynamically imports its backend modules
```
Comment (bwoodsend) [inline news/22.new.rst:1]: Can we have a `.` at the end of the sentence?

Comment (agronholm) [reply news/22.new.rst:1]: Done.

**Comment by bwoodsend** [comment]
@Legorooj I'm happy with this. Your turn to check...

**Comment by Legorooj** [comment]
@agronholm we have Travis, AppVeyor, and GitHub actions because our project lead chose that. Ignore the failing CI; it won't work till the next release of PyInstaller.

**Comment by agronholm** [comment]
Thanks!

**Pull request merged**